Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -1023,7 +996,7 @@ public static bool Is8Dot3(string name)
 
         public static bool Is8Dot3Char(char ch)
         {
-            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%�-{}()@'`&".IndexOf(ch) != -1;
+            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%£-{}()@'`&".IndexOf(ch) != -1;
```

Comment:
My bad, it's the encoding of the £ sign which has changed. You can see it in the GitHub notification e-mails, in the original code the £ is not printed, whereas in the updated code it is.

VS complained that some of the files had inconsistent line endings and also saved everything as UTF-8, so I guess that's why it shows up in the Github diff now.